From ed69f7c667f4fe660584cf466728a46148aee9a2 Mon Sep 17 00:00:00 2001
From: Nicholas Fish <github@seaofdirac.net>
Date: Sat, 21 Jan 2017 12:48:14 +0100
Subject: [PATCH] add missing sys/time.h header

struct timeval is defined in sys/time.h with a musl toolchain.

Fixes:
In file included from Function.cc:16:0:
Calculator.h:241:17: error: field 't_print_end' has incomplete type 'timeval'
  struct timeval t_print_end;
                 ^~~~~~~~~~~
Calculator.h:241:9: note: forward declaration of 'struct timeval'
  struct timeval t_print_end;
         ^~~~~~~
In file included from Calculator.cc:14:0:
Calculator.h:241:17: error: field 't_print_end' has incomplete type 'timeval'
  struct timeval t_print_end;
                 ^~~~~~~~~~~
Calculator.h:241:9: note: forward declaration of 'struct timeval'
  struct timeval t_print_end;
         ^~~~~~~
In file included from DataSet.cc:17:0:
Calculator.h:241:17: error: field 't_print_end' has incomplete type 'timeval'
  struct timeval t_print_end;
                 ^~~~~~~~~~~
Calculator.h:241:9: note: forward declaration of 'struct timeval'
  struct timeval t_print_end;
         ^~~~~~~
In file included from Variable.cc:16:0:
Calculator.h:241:17: error: field 't_print_end' has incomplete type 'timeval'
  struct timeval t_print_end;
                 ^~~~~~~~~~~
---
 libqalculate/Calculator.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libqalculate/Calculator.h b/libqalculate/Calculator.h
index b4cfc57..2346e73 100644
--- a/libqalculate/Calculator.h
+++ b/libqalculate/Calculator.h
@@ -14,6 +14,7 @@
 
 #include <libqalculate/includes.h>
 #include <libqalculate/util.h>
+#include <sys/time.h>
 #include <pthread.h>
 
 /** @file */
-- 
2.11.0

